plugins/isp/internode_usage

```diff
@@ -104,6 +104,25 @@ fetch() {
 		    && false)
 }
 
+get_cached_api_xml() {
+	API_XML=''
+	API_XML_CACHE="${MUNIN_PLUGSTATE}/$(basename "${0}").cache"
```

Comment:
I would suggest to move this up closer to the top of the plugin code. This will make it more visible, that cached data is used.

Reply:
Hum, I prefer the code organised like this, with helper functions together, and data-collecting functions together.

However, I added some details about the caching mechanism in the documentation.

Reply:
I only meant the definition of the variable (`API_XML_CACHE`), as this filename is part of the interface of the plugin. But anyway: personal preferences vary (which is good) ... :)

Reply:
Oh right, sorry. Yeah, it was only one line up, but I see your point. It will happen next time I have improvements to add!